src/polyrith.lean

```diff
@@ -224,7 +244,9 @@ by ring_nf; simp only [complex.I_sq]; ring
 
 -- # main tactic
 
-meta def polyrith : tactic unit :=
+declare_trace polyrith
+
+meta def _root_.tactic.interactive.polyrith : tactic unit :=
```

Comment:
Any tactic used in `begin...end` blocks should be in the `tactic.interactive` namespace.